src/commands/setup-docker.yml

```diff
@@ -6,17 +6,17 @@ parameters:
   docker-username:
     type: env_var_name
     default: DOCKER_USERNAME
-    description: >
+    description: |
       Username for the docker acccount
-
   docker-password:
     type: env_var_name
     default: DOCKER_PASSWORD
-    description: >
+    description: |
       Password for the docker account
-
 steps:
   - setup_remote_docker
   - docker/install-docker
-  - run: |
-    docker login -u << parameters.docker-username >> -p << parameters.docker-password >>
\ No newline at end of file
+  - run:
```

Comment:
I think we are using docker/check here, you can pull it from orbUpdates

Reply:
Yes, we may want to branch off of your other PR that has the setup-docker changes

Reply:
Yes, we may want to branch off of your other PR that has the setup-docker changes